runtime/testy/stateManager.py

```diff
@@ -44,17 +55,15 @@ def start(self):
     # And that there are the correct number of elements
     while True:
       request = self.input.get(block=True)
-      cmd_type = request[0]
```

Comment:
add this back in, along with `arguments = request[1]`. 

We should probably also rename it to python style `cmdType`

Makes the code a lot more readable and clear. Self-documenting code best code